pkg/cmd/experimental/ipfailover/ipfailover.go

```diff
@@ -150,10 +150,18 @@ func Run(f *clientcmd.Factory, options *ipfailover.IPFailoverConfigCmdOptions, c
 		return err
 	}
 
-	if len(options.ServiceAccount) == 0 {
-		return fmt.Errorf("you must specify a service account for the ipfailover pod with --service-account, it cannot be blank")
+	if options.VRRPIDOffset < 0 || options.VRRPIDOffset > 254 {
+		return fmt.Errorf("The vrrp-id-offset must be in the range 0..254")
```

Comment:
Hmm, if we set `iid` to 0 above then it can be an offset from 0 to 255 - this way we lose 1 vrrp id right?